### ISSUE TITLE
Improve Exhibit documentation

### ISSUE THREAD

**Issue by lmorillas** [simile-widgets/exhibit#172]
We should improve and reorganize exhibit documentation.

Old exhibit 2 and exhibit 3 docs are mixed.

Do you have  a plan about it @karger ?

**Comment by karger** [comment]
yes, we've begun some improvements and will be pushing them out soon.

**Comment by lmorillas** [comment]
Cool. Tell us if you want collaboration.

**Comment by karger** [comment]
always!  what do you think you can contribute?

**Comment by jaygray0919** [comment]
Here's an idea.
Use ReadTheDocs (https://readthedocs.org/) and Sphinx as the documentation platform.
Create an Exhibit documentation Github repo.
Use Webhooks to periodically update RTD with new or changed documentation.
Then the community could author pages on the subjects about which they are knowledgeable.
Source format is .rst using docutils (http://docutils.sourceforge.net/) which is a superset of markdown.
@karger should probably set the outline - top level menus.
Contributors like @lmorillas and others active participants on Google Groups then can could work on a menu branch.
Overtime, the best of EX2 and EX3 could be merged with community contributions.
The working version will always be irregular, as with any work-in-progress.
But the Sphinx/.rst documentation services enable multi-media explanation/illustrations.

/jay
